Idno/Core/Config.php

```diff
@@ -182,6 +182,50 @@ function loadIniFiles()
                             }
                         }
                     }
+                    if (array_key_exists('DATABASE_URL', $_ENV)) {
```

Comment:
Hmm... 

If we're going to allow $_ENV variables, which I'm not entirely opposed to, I'd prefer it applied in a slightly more universal way so that we can pass other variables as well. 

I imagine using the KNOWN_ prefix like we already do with $_SERVER...

Reply:
> If we're going to allow $_ENV variables, which I'm not entirely opposed to, I'd prefer it applied in a slightly more universal way so that we can pass other variables as well.

Can you put this another way. I'm not entirely sure what you mean by it.

> I imagine using the KNOWN_ prefix like we already do with $_SERVER...

Is this related to the above? I can absolutely prefix all the ENV, although for some addons it will mean a less straightforward user-experience.

The specific cloud experience this is working towards is a one-click heroku installer. It'll be less turn-key, and more error prone if users have to manually change things

Reply:
It could also benefit docker if that is something you're interested in. I'm trying to keep the shared-hosting ability of this as much as possible.

Reply:
@mapkyca this now pulls everything which can be aliased including a prefix  `KNOWN_`

`CLOUDCUBE` has to be an exception as it does not support deploy-time or automated aliasing sadly.

Reply:
Was thinking more that we'd strip the KNOWN_ off the ENV var so that we can use the same mechanism to set other variables

Reply:
Come again? Above it seems you wanted to prefix. Do you have a more complete vision of what you need to get this into known?